examples/guides/se-restful-webservice/README.adoc

```diff
@@ -880,22 +741,21 @@ Run this `curl` command to retrieve the collected metrics:
 curl -X GET http://localhost:8080/metrics
 ----
 
-You should see a long JSON result. Note two parts:
+You should see a long response. Note two items:
 |====
 |Output |Meaning
 
 a|[listing]
-"application":{"accessctr":4}
+ application:accessctr 4
 
 |The counter we added to the app
 
 a|[listing]
-"requests.meter":{"count":5, ...
+vendor:requests_count 7
 
-|The total HTTP requests the Helidon web server received (and several values
-reflecting the request arrival rate)
+|The total HTTP requests the Helidon web server received
```

Comment:
The total number of HTTP requests that ...

Reply:
done